.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -39,8 +39,9 @@ _Due: <release-deadline>_
 - [ ] Delete Durable Functions packages from myget.
 - [ ] Run Durable Functions release pipeline.
 - [ ] Push myget package to nuget (nuget.org extensions package option).
-- [ ] Create a PR in the [Azure Functions templates repo](https://github.com/Azure/azure-functions-templates) to update templates to the latest version.
-- [ ] Create a PR in the [Azure Functions bundles repo](https://github.com/Azure/azure-functions-extension-bundles) to update bundles to the latest version.
+- [ ] Create a PR in the [Azure Functions templates repo](https://github.com/Azure/azure-functions-templates) targeting branch `v3.x` to update *only the dotnet* templates (`template.json` files) to the latest version.
+- [ ] Create a PR in the [Azure Functions templates repo](https://github.com/Azure/azure-functions-templates) targeting branch `v4.x` to update *only the dotnet* templates (`template.json` files) to the latest version.
+- [ ] Create a PR in the [Azure Functions bundles repo](https://github.com/Azure/azure-functions-extension-bundles) to update bundles to the latest version _if and only if this is a new major release_.
```

Comment:
Can we move the major release condition to the beginning of this point?

```suggestion
- [ ] _if and only if this is a new major release_, Create a PR in the [Azure Functions bundles repo](https://github.com/Azure/azure-functions-extension-bundles) to update bundles to the latest version .
```

Reply:
Yup, good idea! Will commit the suggestion